Example/iOS Example/AppDelegate.swift

```diff
@@ -43,6 +43,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 return
         }
         PiwikTracker.configureSharedInstance(withSiteID: siteid, baseURL: baseurl)
+        PiwikTracker.shared?.logger?.minLevel = .info
```

Comment:
There is a ticket (#118) to ensure the Tracker can be instantiated multiple times and I agree that it is necessary to do so. But I don't think we should remove the default, shared logger. I think most users just use a single tracker and are used to have a shared instance. At the same time it doesn't break any other features.